### ISSUE TITLE
Question about datacamp: payment necessary

### ISSUE THREAD

**Issue by jveldman** [janboone/programming_for_economists#1]
I am an repeater EBE student from tutorial group 6. 
I have a problem with Datacamp. While doing the course "Intermediate python for data science", I receive the message that I need to pay to continue. Although I'm a repeater, I really want to finish Datacamp to ensure I'll pass this year.  What can I do about this?
With kind regards, 

Jelle Veldman
2011313

**Comment by janboone** [comment]
Hi Jelle,

You signed up for datacamp last year and that subscription runs for
about 6 months.

Hence you need to sign up again. Follow the steps under Important on
this page:

https://janboone.github.io/programming_for_economists/pages/

Also the exam will be different this year, hence make sure that you read
this page carefully:

https://janboone.github.io/programming_for_economists/pages/exam/

Regards,

Jan.




jjhv97 <notifications@github.com> writes:

> I am an repeater EBE student from tutorial group 6. 
> I have a problem with Datacamp. While doing the course "Intermediate python for data science", I receive the message that I need to pay to continue. Although I'm a repeater, I really want to finish Datacamp to ensure I'll pass this year.  What can I do about this?
> With kind regards, 
>
> Jelle Veldman
> 2011313


-- 
Jan Boone
http://janboone.github.io/homepage

**Comment by jveldman** [comment]
Hello,

Thank you for your quick response.
I tried making a new account for data-camp,
But since I already used my Tilburg university mail last year, I keep getting the message that the e-mail is already in use.
I signed up with j.j.h.veldman@tilburguniversity.edu<mailto:j.j.h.veldman@tilburguniversity.edu>.
I already did the other things, like the google form etc.

Is there another way to get access to the mandatory courses in Datacamp?

With kind regards,

Jelle Veldman
2011313
U669221
